CONTRIBUTING.rst

```diff
@@ -23,6 +23,10 @@ Contributions
 Contributions are welcome and are greatly appreciated! Every little bit helps,
 and credit will always be given.
 
+This document aims to explain the subject of contributions if you have not contributed to
+any Open Source project, but it will also help people who have contributed to other projects learn about the
+rules of that community.
+
```

Comment:
We should also ad reference to the COMMITTERS.rst

Reply:
Doe

Reply:
I cannot see it ? I thought about referring to "COMMITTERS.rst" from here. Just mention that information about how to be come committer/pmc is is available there. Otherwise no-one will even know that the file exists. CONTRIBUTING.rst is the "starting" point (it is linked to when PR is opened and even tooltip is displayed when it changed).

Reply:
I added this in the section that uses the words PMC and Commiter for the first time.
https://github.com/apache/airflow/pull/13742/files#diff-76e7ca2f2843ac5f659b45382768c1b64bf8704e8d3c6189c4e3623f5d465bc2R103

Reply:
AH. OK. I see it now.